kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java

```diff
@@ -83,6 +111,32 @@ public PatternViewModel() {
                     .addProperty(OTHER_NAME_DESCRIPTION_NAME_TEXT, "")
                     // Ordered collection of Fields
                     .addProperty(FIELDS_COLLECTION, new ArrayList<PatternField>())
+                    .addProperty(IS_INVALID, true)
```

Comment:
Question: Is `IS_INVALID` property bound to the globe button? 
disable or enable based on validation below.

Reply:
no, I'll make that change

Reply:
fixed